core/src/test/scala/org/http4s/otel4s/middleware/ClientMiddlewareTests.scala

```diff
@@ -177,20 +178,28 @@ class ClientMiddlewareTests extends CatsEffectSuite {
             }
 
             val tracedClient = ClientMiddleware.default[IO].build(fakeClient)
-            val request = Request[IO](Method.GET, uri"http://localhost/?#")
```

Comment:
what's the reason for this change?

Reply:
Fewer attributes to test. `url.query` and `url.fragment` will be set to an empty string anyway.
And we test that `url.query` and `url.fragment` are extracted correctly above.